flocx_market/tests/base_test.py

```diff
@@ -0,0 +1,36 @@
+from unittest import TestCase
+from flocx_market.cmd.app import application
+from flocx_market.db.orm import orm
+
+
+class BaseTest(TestCase):
```

Comment:
Not all tests will involve the database, so I might have the BaseTest just set the context, and have something like BaseDBTest that extends BaseTest and which sets up the database.

Reply:
Hey, I only use BaseTest for integration and System tests which usually involve some kind of crud operation. For unit tests, I use UnitBaseTest which doesn't involve the DB. 
What do you think about this approach @tzumainn ?

Reply:
Ah, okay - that's fair - thanks!